### PR TITLE
Makefile: add 'test-fast', change 'test' to 'vim/test' and make 'test'

### PR DESCRIPTION
**Pull request by tyru**
* Add `test-fast` task
* Change `test` to `vim/test` and now `make test` runs all test tasks

### CODE REVIEW

**Comment by tyru** [comment]
This change allows developers **to run the same tests on travis ci easily.** (`make test`)
before this change, complete test tasks are managed in `.travis.yml`

File: Makefile [hunk 1]
```diff
@@ -22,8 +25,7 @@ checkpy: all
 checkvim: all
 	vint autoload
 
-test:
-	test/run.sh
+vim/test:
```
Comment (blueyed) [inline Makefile]: This looks wrong.. should run `test/run.sh` still probably?

Comment (tyru) [reply Makefile]: oops... fixed.

Comment (tyru) [reply Makefile]: ah also `fast-test` must reference this task...

**Comment by tyru** [comment]
Thanks :tada:

**Pull request merged**